### PR TITLE
Stop overruling interpreter in updategrub.py

### PR DESCRIPTION
**Pull request by ydirson**
The script requires python3 now, forcing python2 naturally causes errors.

### CODE REVIEW

**Comment by stormi** [comment]
Do you think we should also explicitly `Require` python3, despite the package still has python in its BuildRequires?

**Comment by ydirson** [comment]
> Do you think we should also explicitly `Require` python3, despite the package still has python in its BuildRequires?

Thought about it, but that's rather the job of the package providing the script, and it does that already.

File: SPECS/kernel-alt.spec [hunk 1]
```diff
@@ -1047,8 +1047,11 @@ fi
 %{python2_sitearch}/*
 
 %changelog
+* Thu Apr 04 2024 Yann Dirson <yann.dirson@vates.tech> - 4.19.227-6
+- Stop overruling interpreter in updategrub.py
+
 * Mon Feb 05 2024 Yann Dirson <yann.dirson@vates.tech> - 4.19.227-5
-- use updategroup.py from /opt/xensource/bin
+- Use updategroup.py from /opt/xensource/bin
```
Comment (stormi) [inline SPECS/kernel-alt.spec]:
```suggestion
- Use updategrub.py from /opt/xensource/bin
```

**Comment by stormi** [comment]
Koji build to v8.3-incoming: http://koji.xcp-ng.org/taskinfo?taskID=49865

**Pull request merged**